net/knot/Makefile

```diff
@@ -144,6 +144,8 @@ define Package/knot-tests/description
 	Usage: /usr/share/knot/runtests.sh
 endef
 
+export KNOT_VERSION_FORMAT=release
```

Comment:
Why do we need this?

Reply:
Since Knot DNS 2.8.0 if the build system detects git is present and the last commit is not tagged, extended version is used (e.g. 2.8.1555068070.3bb49ae). In combination with openwrt packages, which are in completely different git, the server shows confusing version.